tests/test_queryset.py

```diff
@@ -46,6 +46,13 @@ def test_df():
     assert set(df["features__name"].iloc[0]) == set(feature_names)
     assert set(df["features__created_by_id"].iloc[0]) == {current_user_id()}
 
+    # join parents on features
+    df = ln.ULabel.filter().df(include=["features__name", "features__created_by_id"])
```

Comment:
Thank you for this test!

Just: This doesn't test the `join = "outer"` option that you added. Can you make a small modification that actually makes the call using this parameter? Feel free to use the ame example!

Reply:
Yes, planning to add this!